docs/index.md

```diff
@@ -21,12 +21,12 @@ Autonolas' mission is to provide a foundation on which new types of autonomous a
 
 ## The Autonolas Ecosystem
 
-The Autonolas Ecosystem is composed of three main components:
+The Autonolas Stack is made of three main components:
```

Comment:
The title is "Autonolas Ecosystem" though (?)

Also, Autonolas ecosystem is mentioned below in line 32.

Reply:
Changed that occurence.